ircbot.py

```diff
@@ -63,8 +63,21 @@ def _showError(self, failure):
     def command_live(self, rest):
         player = rest.partition(' ')
         if player:
-            # find the player on lichess
-            pass
+            try:
+                f = urllib.request.urlopen("http://en.lichess.org/api/user/" + username).read().decode("utf-8")
```

Comment:
can we rename f to a more obvious variable? in this case something like 'response'.